tools/peptideshaker/searchgui.xml

```diff
@@ -184,139 +346,534 @@
+			                	<option value="0" >0</option>
+			                	<option value="10" >10</option>
+			                	<option value="20" >20</option>
+			                	<option value="30" selected="True">30</option>
+			                	<option value="40" >40</option>
+			                	<option value="50" >50</option>
+			                	<option value="60" >60</option>
+	</param>
+
+                <param name="tide_monoisotopic" type="boolean" truevalue="1" falsevalue="0" checked="true"
+                    label="TIDE: Monoisotopic" help="If true, the precursor mass is monoisotopic"/>
+                <param name="tide_clip_n_term" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Clip Nterm Methionine" help="If true, the Nterm Methionine will be clipped"/>
+  	<param name="tide_digestion_type" label="TIDE: Digestion Type" type="select" help="Either both ends (full-digest) or at least one end (partial-digest) of a peptide must conform to enzyme specificity rules">
+			                	<option value="full-digest" selected="True">full-digest</option>
+			                	<option value="partial-digest" >partial-digest</option>
+	</param>
+                <param name="tide_compute_sp" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Compute SP" help="If true, the SP-score is calculated"/>
+                <param name="tide_max_psms" type="integer" value="10"
+                    label="TIDE: Maximum Number of PSMs" help="Set the maximum number of PSMs to be considered"/>
+                <param name="tide_compute_p" type="boolean" truevalue="1" falsevalue="0" checked="false"
+                    label="TIDE: Compute Exact P-value" help="If true, the exact p-values are calculated"/>
+                <param name="tide_min_spectrum_mz" type="float" value="0.0"
+                    label="TIDE: Minimum Spectrum m/z" help="Set the minimum spectrum m/z value for a spectrum to be considered"/>
+                <param name="tide_max_spectrum_mz" type="float" value="100000.0"
+                    label="TIDE: Maximum Spectrum m/z" help="Set the maximum spectrum m/z value for a spectrum to be considered"/>
+                <param name="tide_min_spectrum_peaks" type="integer" value="20"
+                    label="TIDE: Minimum Spectrum Peaks" help="Set the minimum amount of peaks in a spectrum for it to be considered"/>
+  	<param name="tide_spectrum_charges" label="TIDE: Spectrum Charges" type="select" help="Select what precursor charges should be taken into account for matching">
+			                	<option value="1" >1</option>
+			                	<option value="2" >2</option>
+			                	<option value="3" >3</option>
+			                	<option value="all" selected="True">all</option>
+	</param>
+                <param name="tide_remove_prec" type="boolean" truevalue="1" falsevalue="0" checked="false"
```

Comment:
You can handle these booleans a lot easier if you put the actual parameter into the `truevalue`. 
e.g. `truevalue="-tide_remove_prec_tol 1" falsevalue=""`

In the `command` tag you simply specify the variable: `${tide.tide_remove_prec_tol}`